examples/nodejs-simple/Dockerfile

```diff
@@ -14,9 +14,24 @@
 
 FROM debian:buster
 RUN useradd -u 1000 -m server
-RUN apt-get update && apt-get install -y curl  && apt-get clean
-RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
-    apt-get install -y nodejs
+RUN apt-get update && apt-get install -y curl software-properties-common gnupg  && \
```

Comment:
do we still need to install curl if we aren't running curl | bash anymore?

Reply:
We still need the curl since we use the curl in line 27 to get the key but we won't use curl with bash any more.